localization/xliff/vscode-mssql.xlf

```diff
@@ -1,4684 +1,4687 @@
-    </trans-unit>
-    <trans-unit id="mssql.walkthroughs.getStarted.title">
-      <source xml:lang="en">Get Started with MSSQL for Visual Studio Code</source>
-    </trans-unit>
-    <trans-unit id="mssql.showGettingStarted">
-      <source xml:lang="en">Getting Started Guide</source>
-    </trans-unit>
-    <trans-unit id="mssql.copilot.editorSubmenu">
-      <source xml:lang="en">MSSQL Copilot</source>
-    </trans-unit>
-    <trans-unit id="mssql.Configuration">
-      <source xml:lang="en">MSSQL configuration</source>
-    </trans-unit>
-    <trans-unit id="mssql.walkthroughs.getStarted.connectToDatabase.description">
-      <source xml:lang="en">Make a new connection to a SQL database, or edit existing connections with the connection dialog.&#10;You can connect to a database by entering your connection information, using a connection string, or browsing your Azure subscriptions.&#10;[Open Connection Dialog](command:mssql.addObjectExplorer)</source>
-    </trans-unit>
-    <trans-unit id="mssql.manageProfiles">
-      <source xml:lang="en">Manage Connection Profiles</source>
-    </trans-unit>
-    <trans-unit id="mssql.query.maxXmlCharsToStore">
-      <source xml:lang="en">Maximum number of characters to store for each value in XML columns after running a query. Default value: 2,097,152. Valid value range: 1 to 2,147,483,647.</source>
-    </trans-unit>
-    <trans-unit id="mssql.query.maxCharsToStore">
-      <source xml:lang="en">Maximum number of characters/bytes to store for each value in character/binary columns after running a query. Default value: 65,535. Valid value range: 1 to 2,147,483,647.</source>
-    </trans-unit>
-    <trans-unit id="mssql.logFilesRemovalLimit">
-      <source xml:lang="en">Maximum number of old files to remove upon startup that have expired mssql.logRetentionMinutes. Files that do not get cleaned up due to this limitation get cleaned up next time Azure Data Studio starts up.</source>
-    </trans-unit>
-    <trans-unit id="mssql.query.setRowCount">
-      <source xml:lang="en">Maximum number of rows to return before the server stops processing your query.</source>
-    </trans-unit>
-    <trans-unit id="mssql.query.textSize">
-      <source xml:lang="en">Maximum size of text and ntext data returned from a SELECT statement</source>
-    </trans-unit>
-    <trans-unit id="mssql.newDeployment">
-      <source xml:lang="en">New Deployment</source>
-    </trans-unit>
-    <trans-unit id="mssql.objectExplorerNewQuery">
-      <source xml:lang="en">New Query</source>
-    </trans-unit>
-    <trans-unit id="mssql.newQuery">
-      <source xml:lang="en">New Query</source>
-    </trans-unit>
-    <trans-unit id="mssql.newTable">
-      <source xml:lang="en">New Table</source>
-    </trans-unit>
-    <trans-unit id="mssql.walkthroughs.nextSteps.title">
-      <source xml:lang="en">Next Steps with MSSQL for Visual Studio Code</source>
-    </trans-unit>
-    <trans-unit id="mssql.logRetentionMinutes">
-    
```

Comment:
revert this file; if we need to fix the line endings, that should be done in a separate PR